docs/embedding/sdk/quickstart-cli.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Embedded analytics SDK - CLI quickstart
+---
+
+# Embedded analytics SDK - CLI quickstart
+
+{% include beta-blockquote.html %}
+
+We built a single command to spin up a Metabase and help you get an embedded dashboard in your app. This setup won't work in production; it's only intended for you to quickly try out the SDK on your local machine.
+
+## Prerequisites
+
+- Docker
+- [Node.js 18.x LTS](https://nodejs.org/en) or higher.
+- License (Optional - only if you want to try out multi-tenancy).
+- Database (you can connect to your app's database).
+
+## The quickstart command
+
+Change into your React application and run:
+
+```sh
+npx @metabase/embedding-sdk-react@latest start
+```
+
+## Script overview
+
+The script will walk you through the setup. There are a fair number of pieces to put together, so here's an overview of what the command does.
+
+1. Checks that you're installing the SDK in a React application.
+2. Looks for, or installs, the Embedded Analytics SDK.
```

Comment:
nit: I see 3 kinds of capitalizations of "embedding analytics sdk" in this PR. I think this one should probably be "Embedded analytics SDK" to match the others?

- Embedded Analytics SDK
- Embedded analytics SDK
- embedded analytics sdk